src/WebUI/Common/KubeSummary.tsx

```diff
@@ -203,7 +204,7 @@ export class KubeSummary extends BaseComponent<IKubeSummaryProps, IKubernetesCon
                     title={this.props.title}
                     titleSize={TitleSize.Large}
                     className={"content-main-heading"}
-                    description={localeFormat(Resources.NamespaceHeadingText, this.state.namespace || "")}
+                    description={this.props.clusterName ? localeFormat(Resources.SummaryHeaderSubTextFormat, this.props.clusterName) : undefined}
```

Comment:
i am thinking, if we do not have cluster name, show namespace. what ever text we are showing today.